### DISCUSSION TITLE
attempt to extend manual with FM broadcast

### DISCUSSION THREAD

**Discussion by gartnl** [egzumer/uv-k5-firmware-custom#251] [Show and tell]
Below an attempt for a description of the broadcast FM mode. If Egzumer/Erik are OK with it I would like to incorporate it into the main manual.

====================

**BROADCAST FM**

The radio is capable of receiving broadcast FM from 76 to 108 MHz. For this it uses a separate chip. RDS is not implemented.

_During broadcast reception the active VFO still has priority. This means that any reception on the active VFO will temporarily disable broadcast reception and the radio switches to VFO reception. At the end of VFO reception the radio switches back to broadcast._

**_BASIC OPERATION:_**

o	‘F-FM’ or long press ‘FM’ starts broadcast reception
o	‘Exit’, 'F-FM' or long press 'FM' ends broadcast reception
o	‘F-VFO/MR’ or long press ‘VFO/MR’ changes between VFO or memory channels

**Set a frequency in FM-VFO mode:** simply typing a frequency sets it to receive. Resolution is 100 kHz so entering 929 would tune to 92.9 MHz. Use arrow keys to change in 100 kHz steps.
**Store in memory from FM-VFO mode:** pressing ‘M’ in VFO mode allows to store the current  frequency in a memory channel. Use arrow keys to select memory, confirm with ‘M’. There are 20 memories available.
**Delete a stored memory:** in MR mode pressing ‘M’ allows to delete that memory channel.
**Select a memory:** in MR mode entering 01-20 selects a memory channel. Use arrow keys to step a channel up or down.

**_SCANNING FOR STATIONS FROM FM-VFO:_**

**Auto:** start with ‘F-\*’ or long press ‘\*’
The radio scans for stations and stores the first 20 stations in memory. Scanning starts at the low side of the band. Initiating auto-scan will delete previously stored channels. ‘Exit’ ends autoscan.
**Manual:** Short press ‘\*’ starts manual scan. The radio scans from the current frequency up until a station is received. You can continue scanning in any direction using arrow keys. ‘Exit’ stops scanmode.

===============

**Comment by egzumer** [comment]
I added it myself where I wanted it to be, also fixed formatting.
https://github.com/egzumer/uv-k5-firmware-custom/wiki/FM-broadcast-radio-receiver

If you want to rearrange it go ahead, just use proper markdown formatting.

**Reply by gartnl** [reply]
Great! Thanks. Might add the 'FM' display sign available in the new firmware later. But now it has a propper starting place to work from.

**Comment by ErikS-web** [comment]
Nice info and chapter added in the overview
